packages/design-system/src/components/Tag/Tag.tsx

```diff
@@ -1,12 +1,35 @@
 import React from 'react';
 
-import * as S from './Tag.style';
+import Primitive from './primitive';
+import { TagDefault, TagInformation, TagSuccess, TagWarning, TagDestructive } from './variations';
 
-export type TagProps = React.PropsWithChildren<any>;
+export enum TagVariant {
+	Default = 'Default',
```

Comment:
```suggestion
```
variant is optional, we don't need to create this enum value

Reply:
It's used to have autocomplete actually because Default is a standalone variant